lib/matplotlib/axes/_subplots.py

```diff
@@ -187,6 +187,18 @@ def _make_twin_axes(self, *kl, **kwargs):
         return ax2
 
 
+# this here to support cartopy which was using a private part of the
+# API to register their Axes subclasses.
+
+# In 3.1 this should be changed to a dict subclass that warns on use
```

Comment:
Because there is not a released version of cartopy with the fix on their side yet.  I don't want to warn at there users and give them no way to fix it.